smart_campus/app/templates/app/group_list_page.html

```diff
@@ -0,0 +1,71 @@
+{% extends "app/base.html" %}
+{% load static %}
+
+{% block title %}Smart Campus{% endblock %}
```

Comment:
Is this title block necessary?
If I'm not misunderstood, this is already defined in `base.html`.

Reply:
Yes, this is the duplicated block.